assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidgetV2/UserDimensionsPieChart.js

```diff
@@ -89,7 +89,7 @@ export default function UserDimensionsPieChart( { dimensionName, entityURL, sour
 							FORM_ALL_TRAFFIC_WIDGET,
 							{
 								dimensionValue: __( 'Others', 'google-site-kit' ) === dimensionValue ? '' : dimensionValue,
-								dimensionColor: slices[ row ]?.color,
+								dimensionColor: __( 'Others', 'google-site-kit' ) === dimensionValue ? '' : slices[ row ]?.color,
```

Comment:
@johnPhillips could you please assign `__( 'Others', 'google-site-kit' ) === dimensionValue` to a variable and use it for both values to escape duplicates? Something like this would work:

```javascript
const isOthers = __( 'Others', 'google-site-kit' ) === dimensionValue;
setValues(
	FORM_ALL_TRAFFIC_WIDGET,
	{
		dimensionValue: isOthers ? '' : dimensionValue,
		dimensionColor: isOthers ? '' : slices[ row ]?.color,
	}
);
```

Reply:
Of course, don't know why I didn't do that first time lol